lib/mobility-core/src/Kernel/External/Notification/FCM/Flow.hs

```diff
@@ -128,7 +129,8 @@ createAndroidNotification title body notificationType =
             Just $
               FCMNotificationIconUrl
                 "http://localhost:8080/static/images/ride-success.png",
-          fcmdTag = Just notificationType
+          fcmdTag = Just notificationType,
+          fcmdSound = sound
```

Comment:
this will override fcmSound for `ALLOCATION_REQUEST` and `TRIP_STARTED` case as well. do we want that ?

Reply:
No, these sound names are not even in the code and for TRIP_STARTED there is other sound which is handled in android side

Reply:
then should we remove this code itself ?

```
let notification = case notificationType of
        ALLOCATION_REQUEST ->
          def{fcmdSound = Just "notify_sound.mp3",
              fcmdChannelId = Just "RINGING_ALERT"
             }
        TRIP_STARTED ->
          def{fcmdSound = Just "notify_otp_sound.mp3",
              fcmdChannelId = Just "TRIP_STARTED"
             }
        _ -> def
 ```

Reply:
haan yes, we can remove this